getting-started/templates/systemlink-values.yaml

```diff
@@ -508,6 +508,12 @@ fileingestion:
     ## S3 Port
     ##
     port: *minioPort
+  ## Configure rate limiting per user per replica over one second.
```

Comment:
I'm pretty sure this can be better formulated, so feedback is appreciated

Reply:
```suggestion
   ## Configure rate limiting. Limits are enforced per-user.  Each replica of the file ingestion service 
   ## applies its own per-user limit. With load-balancing, the effective rate will be higher than the 
   ## individual rates configured here. 
```

Reply:
If you expect the user to need to modify this value for an average deployment, include an <ATTENTION> tag here.

Reply:
I think `3` is already a reasonable limit. The only use-case I can think of is when we want to migrate a lot of data at once

Reply:
do you mean adding `<ATTENTION>`?

Reply:
@JD-Robertson, to answer your question:
> If this is a new setting and something that the user is likely to want to modify, consider adding it to this document so it will be included in the release notes.

This is an old setting we just didn't document it. So I don't think we have to add it to the document